src/hls-adapter.js

```diff
@@ -136,6 +136,15 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
     if (Utils.Object.hasPropertyPath(config, 'playback.useNativeTextTrack')) {
       adapterConfig.subtitleDisplay = Utils.Object.getPropertyPath(config, 'playback.useNativeTextTrack');
     }
+    if (Utils.Object.hasPropertyPath(config, 'abr.fpsDroppedFramesInterval')) {
+      adapterConfig.fpsDroppedFramesInterval = config.playback.fpsDroppedFramesInterval;
```

Comment:
need to pass this and `fpsDroppedMonitoringThreshold`, capLevelOnFPSDrop via the hlsjs config, e.g. `adapterConfig.hlsConfig.xyz = ...`

Reply:
and move them to `config.abr`